source/bson-corpus/tests/double.json

```diff
@@ -5,65 +5,77 @@
     "valid": [
         {
             "description": "+1.0",
-            "bson": "10000000016400000000000000F03F00",
-            "extjson": "{\"d\" : {\"$numberDouble\": \"1.0\"}}"
+            "canonical_bson": "10000000016400000000000000F03F00",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.0\"}}",
+            "relaxed_extjson": "{\"d\" : 1.0}"
         },
         {
             "description": "-1.0",
-            "bson": "10000000016400000000000000F0BF00",
-            "extjson": "{\"d\" : {\"$numberDouble\": \"-1.0\"}}"
+            "canonical_bson": "10000000016400000000000000F0BF00",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"-1.0\"}}",
+            "relaxed_extjson": "{\"d\" : -1.0}"
         },
         {
             "description": "+1.0001220703125",
-            "bson": "10000000016400000000008000F03F00",
-            "extjson": "{\"d\" : {\"$numberDouble\": \"1.0001220703125\"}}"
+            "canonical_bson": "10000000016400000000008000F03F00",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.0001220703125\"}}",
+            "relaxed_extjson": "{\"d\" : 1.0001220703125}"
         },
         {
             "description": "-1.0001220703125",
-            "bson": "10000000016400000000008000F0BF00",
-            "extjson": "{\"d\" : {\"$numberDouble\": \"-1.0001220703125\"}}"
+            "canonical_bson": "10000000016400000000008000F0BF00",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"-1.0001220703125\"}}",
+            "relaxed_extjson": "{\"d\" : -1.0001220703125}"
         },
         {
-            "description": "+2.0001220703125e10",
-            "bson": "1000000001640000807ca1a9a0124200",
-            "extjson": "{\"d\" : {\"$numberDouble\": \"2.0001220703125e10\"}}"
+            "description": "1.23456789012345677E+18",
+            "canonical_bson": "1000000001640081E97DF41022B14300",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.23456789012345677E+18\"}}",
+            "relaxed_extjson": "{\"d\" : 1.23456789012345677E+18}"
```

Comment:
The spec says:
> String representation of doubles is fairly unportable so it's hard to provide
a single string that all platforms/languages will generate.  Testers may
need to normalize/modify the test cases.

I think that's sufficient for now.